docs/Snowflake-batchsource.md

```diff
@@ -90,7 +90,7 @@ Data Types Mapping
 | TIMESTAMP                      | timestamp/string      | Alias for one of the TIMESTAMP variations (TIMESTAMP_NTZ by default).|
 | TIMESTAMP_LTZ                  | timestamp             | TIMESTAMP with local time zone; time zone, if provided, is not stored.|
 | TIMESTAMP_NTZ                  | timestamp             | TIMESTAMP with no time zone; time zone, if provided, is not stored.|
-| TIMESTAMP_TZ                   | string                | TIMESTAMP with time zone.                                 |
+| TIMESTAMP_TZ                   | timestamp             | TIMESTAMP with time zone.                                 |
```

Comment:
was this mistakenly in the doc before or was it actually being read as a string?

If it was being read as a string, we can't just change the data type as that will break backwards compatibility.

Reply:
it was added mistakenly only, was giving validation error on this data type.